docs/primer/openfibredata.md

```diff
@@ -0,0 +1,32 @@
+Open fibre data
+============
+
+
+## What is open fibre data?
+
+Open data is data that can be freely used, modified, and shared by anyone for any purpose.
+
+Open fibre data describes fibre optic communication networks. It can cover:
+- Spatial data, such as the physical location of the links and nodes in a fibre optic communication network
+- Technical data, such as the number and type of optical fibres in a link
+- Administrative data, such as the organisations that own passive infrastructure or operate active network infrastructure
+
+## Why publish open fibre data?
+
+Open fibre data is useful to many different stakeholders and can be used for many different purposes; for example:
+- Infrastructure and network providers can use data to avoid overbuild, reduce commercial risk and identify existing networks to connect to.
+- Academics, researchers and policy analysts can use data to understand the economic and/or social impact of investing in fibre infrastructure.
+- National statistics offices, regulatory agencies and inter-governmental organisations can use data to calculate internationally agreed indicators.
+- Government, private and multi-lateral investors can use data to identify opportunities to invest in fibre infrastructure.
+- Non-governmental organisations can use data to advocate for improved internet access.
+
+```{note}
+**Maps and data: What’s the difference?**
+
+Maps are a type of data visualisation; they are distinct from the data itself.
+
+Network providers, regulators and others often publish network maps without publishing the underlying data. Lack of access to the underlying data presents challenges for data users; in particular, for users that want to combine data on different networks or join-up data with other sources of social or economic data.
+
+Publishing open fibre data, rather than only maps, reduces barriers to data use and ensures that permission is granted for reuse data. 
+
+```
```

Comment:
Move title. Indent content.

````suggestion
```{note} Maps and data: What’s the difference?
   Maps are a type of data visualisation; they are distinct from the data itself.

   Network providers, regulators and others often publish network maps without publishing the underlying data. Lack of access to the underlying data presents challenges for data users; in particular, for users that want to combine data on different networks or join-up data with other sources of social or economic data.
  
   Publishing open fibre data, rather than only maps, reduces barriers to data use and ensures that permission is granted for reuse data. 
```
````